File: backend/src/main/java/com/sww/ddorangddorang/domain/guess/entity/Guess.java
package com.sww.ddorangddorang.domain.guess.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import java.time.LocalDateTime;
import lombok.AccessLevel;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Entity
public class Guess {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;                    // BIGINT
    private Long participantId;           // BIGINT "현재 플레이어"
    private Long manito;             // BIGINT "추측한 사람"
    private Boolean isCorrect = false;    // TINYINT "정답 여부"
    private LocalDateTime deletedAt;    // TIMESTAMP "삭제 시간"

    public void updateGuessInfo(Long manito, Boolean isCorrect){
        this.manito = manito;
        this.isCorrect = isCorrect;
    }
}

File: frontend/src/modules/guess.ts
import {atom} from 'recoil';
import {GuessInfo} from '../types/guess';

const KEY = 'GUESS';

export const GuessInfoState = atom<GuessInfo>({
  key: `${KEY}/guessinfo`,
  default: {
    id: 0,
    participantId: 0,
    manito: 0,
    isCorrect: true,
    deletedAt: '',
  },
});
